tests/test_updatediff.py

```diff
@@ -554,6 +554,80 @@ def test_skip_update(tmp_path_factory: pytest.TempPathFactory) -> None:
     assert not (dst / "skip_me.rej").exists()
 
 
+def test_skip_update_deleted(tmp_path_factory: pytest.TempPathFactory) -> None:
```

Comment:
Could you parametrize this test to also test files with special characters that need to be escaped internally for `exclude` to make sure they are actually escaped?